apps/web/src/store/spaces.ts

```diff
@@ -0,0 +1,136 @@
+import { create } from 'zustand';
+
+export type TSidebarView = 'close' | 'peers';
+
+export enum TokenGateCondition {
+  HAVE_A_LENS_PROFILE = 'HAVE_PROFILE',
+  FOLLOW_A_LENS_PROFILE = 'FOLLOW_PROFILE',
+  COLLECT_A_POST = 'COLLECT_POST',
+  MIRROR_A_POST = 'MIRROR_POST'
+}
+
+export enum MusicTrack {
+  DEFAULT = 'DEFAULT',
+  CALM_MY_MIND = 'CALM_MY_MIND',
+  CRADLE_OF_SOUL = 'CRADLE_OF_SOUL',
+  FOREST_LULLABY = 'FOREST_LULLABY'
+}
+
+interface SpacesState {
+  showSpacesLobby: boolean;
+  setShowSpacesLobby: (showSpacesLobby: boolean) => void;
+  showSpacesWindow: boolean;
+  setShowSpacesWindow: (showSpacesWindow: boolean) => void;
+  lensAccessToken: string;
+  setLensAccessToken: (lensAccessToken: string) => void;
+  isRecordingOn: boolean;
+  setIsRecordingOn: (isRecordingOn: boolean) => void;
+  isTokenGated: boolean;
+  setIsTokenGated: (isTokenGated: boolean) => void;
+  tokenGateConditionType: TokenGateCondition;
+  setTokenGateConditionType: (
+    tokenGateConditionType: TokenGateCondition
+  ) => void;
+  tokenGateConditionValue: string;
+  setTokenGateConditionValue: (tokenGateConditionValue: string) => void;
+  myReaction: string;
+  setMyReaction: (myReaction: string) => void;
+  space: {
+    id: string;
+    host: string;
+    title: string;
+  };
+  setSpace: (space: { id: string; host: string; title: string }) => void;
+  sidebar: {
+    isSidebarOpen: boolean;
+    sidebarView: TSidebarView;
+  };
+  setSidebarView: (val: TSidebarView) => void;
+  isMyHandRaised: boolean;
+  setMyHandRaised: (val: boolean) => void;
+  myMusicTrack: MusicTrack;
+  setMyMusicTrack: (val: MusicTrack) => void;
+  isMyMusicPlaying: boolean;
+  setIsMyMusicPlaying: (val: boolean) => void;
+  requestedPeers: string[];
+  addRequestedPeers: (val: string) => void;
+  removeRequestedPeers: (val: string) => void;
+  spacesTimeInHour: string;
+  setSpacesTimeInHour: (val: string) => void;
+  spacesTimeInMinute: string;
+  setSpacesTimeInMinute: (val: string) => void;
+  isSpacesTimeInAM: boolean;
+  setIsSpacesTimeInAM: (val: boolean) => void;
```

Comment:
setIsSpacesTimeInAM: (isSpacesTimeInAM: boolean) => void;

change it like this